corehq/apps/domain/templates/domain/manage_releases_by_app_profile.html

```diff
@@ -49,7 +51,11 @@
           <td data-bind="text: version"></td>
           <td data-bind="text: status"></td>
           <td>
-            <button type="button" class="btn default-btn"
+            <a type="button" class="btn btn-default" data-bind="attr: { href: hostedCCZLink },
+              visible: hostedCCZLink" target="_blank">
+              {% trans 'View Hostings' %}
+            </a>
+            <button type="button" class="btn"
```

Comment:
This one should also have `btn-default`

Reply:
i thought there should be only one btn-default in a form/page, so we should always have `btn btn-default` for all buttons?

Reply:
Right idea, wrong specific classes :)  There should be at most one `btn-primary` per page. But every button needs to have both `btn` and a `btn-foo` class (usually the latter is `btn-default`), or else it won't have the right shape to look like a button.

Reply:
all about buttons: https://www.commcarehq.org/styleguide/molecules/#molecules-buttons